webpack.config.js

```diff
@@ -47,9 +47,9 @@ var config = {
     ],
     'output': {
         path: './dist/',
-        filename: 'reactapp/js/[name].app.js',
-        chunkFilename: 'reactapp/js/[name].chunk.js',
-        cssFilename: 'reactapp/css/app.css',
+        filename: 'reactapp/[name].app.js',
+        chunkFilename: 'reactapp/[name].chunk.js',
+        cssFilename: 'reactapp/app.css',
```

Comment:
As mentioned above, maybe better to keep current directory structure to prevent having to update parent project, but again your call.